source/js/Router.js

```diff
@@ -5,114 +5,128 @@
      */
-  navigate(page, statePopped) {
-    console.log('');
-    console.log('');
-    console.log(`navigate() function called, requested page: ${page}`);
-    /**
-      * TODO - Part 1 - Step 4
-      * Now, we are going to call the functions that we stored earlier based on
-      * what page is being requested. For this function:
-      *
-      *  1. First, check to see if the function exists, if it doesn't log an error
-      *     and return out of the function. 'this' is a global variable, so you can
-      *     check to see if it exists nearly the same way you assigned it
-      *  2. Create a variable called hash. If page == 'home' set hash to be an empty
-      *     string, if page is anything else set it to be the string '#' + page, e.g.
-      *     '#ghostCookies'
-      *  3. Next, if statePopped is false and window.location.hash does NOT match the
-      *     hash that you just made, use history.pushState() to add the current state
-      *     and URL + hash to history
-      *  4. Finally, call the stored function for the given page
-      */
-
-    // self-coded:
-    // 1.
-    console.log(`number of pages/properties of this: ${Object.keys(this).length}`);
-    console.log("this's keys:");
-    console.log(Object.keys(this));
-    console.log(`Requested page's function value: ${this[page]}`);
-    // eslint-disable-next-line no-unused-vars
-    // page doesn't exist
-    if (typeof this[page] !== 'function') { // note to self that checking if this[page] == undefined returns true even if the function this[page] is defined
-      console.log('ERROR note to self: requested function for page: ', page, ' does not exist');
-      return;
-    }
+    navigate(page, statePopped) {
+      console.log("");
+      console.log("");
+      console.log(`navigate() function called, requested page: ${page}`);
+      /**
+       * TODO - Part 1 - Step 4
+       * Now, we are going to call the functions that we stored earlier based on 
+       * what page is being requested. For this function:
+       * 
+       *  1. First, check to see if the function exists, if it doesn't log an error
+       *     and return out of the function. 'this' is a global variable, so you can 
+       *     check to see if it exists nearly the same way you assigned it
+       *  2. Create a variable called hash. If page == 'home' set hash to be an empty
+       *     string, if page is anything else set it to be the string '#' + page, e.g.
+       *     '#ghostCookies'
+       *  3. Next, if statePopped is false and window.location.hash does NOT match the
+       *     hash that you just made, use history.pushState() to add the current state
+       *     and URL + hash to history
+       *  4. Finally, call the stored function for the given page
+       */
+  
+      // self-coded:
+      // 1.
+      /*
+      console.log("number of pages/properties of this: " + Object.keys(this).length);
```

Comment:
Create a GitHub Issue for removing these commented out console log statements.

Reply:
sure, #103.